microdnf/commands/download/download.cpp

```diff
@@ -51,10 +51,10 @@ void CmdDownload::set_argument_parser(Context & ctx) {
     download->set_named_args_help_header("Optional arguments:");
     download->set_positional_args_help_header("Positional arguments:");
     download->set_parse_hook_func([this, &ctx](
-                               [[maybe_unused]] ArgumentParser::Argument * arg,
-                               [[maybe_unused]] const char * option,
-                               [[maybe_unused]] int argc,
-                               [[maybe_unused]] const char * const argv[]) {
+                                      [[maybe_unused]] ArgumentParser::Argument * arg,
+                                      [[maybe_unused]] const char * option,
+                                      [[maybe_unused]] int argc,
+                                      [[maybe_unused]] const char * const argv[]) {
```

Comment:
I'm not sure where this indent change is supposed to align? I think we agreed to just use single 4-space indents instead of these long ones (that you need to keep updating whenever the top line changes, but then again the change here seems just unrelated?).